### PR TITLE
Add devtools tests

### PR DESCRIPTION
**Pull request by Piinks**
Add the flutter/devtools test suite for checking against changes in flutter/flutter

Fixes https://github.com/flutter/devtools/issues/4433

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by Piinks** [comment]
For this, I will need to write a custom script in devtools to run the tests. I will follow up on this tomorrow

**Review by kenzieschmoll — APPROVED** [review]

File: registry/flutter_devtools.test [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Tests from https://github.com/flutter/devtools
+
+contact=kenzieschmoll@google.com
```
[the remark this answers is not in the document]
Comment (Piinks) [reply registry/flutter_devtools.test:3]: Definitely, I'll update this.

**Comment by Piinks** [comment]
Hmm... Looks like a bunch of tests are timing out. and not all of the expected mocks are created by the generate script.

**Comment by Piinks** [comment]
Closing since the devtools tests aren't working in this environment as expected right now.

**Pull request closed without merging**